.github/workflows/hashes.yml

```diff
@@ -21,9 +21,9 @@ jobs:
       with:
         repository: "rwth-i6/sisyphus"
         path: "sisyphus"
-    - uses: actions/setup-python@v4
+    - uses: actions/setup-python@v6
       with:
-        python-version: 3.8
+        python-version: 3.10.12
```

Comment:
Q:
1. What is required from 3.10 that requires this version bump?
2. Not that it's necessary for this PR (if you don't import this module), but should 3.10 then become the new minimally required version for `i6_core`? I vaguely remember the time we bumped the python version on apptek_asr that i6_core wasn't ready for a bump yet/there are people using it with older pythons than 3.10. I think this could still be the case.

Reply:
I talked to the people at i6 who probably have the oldest setups (@Marvin84 @mmz33 @JackTemaki ) and they are all OK with bumping the required version to 3.10.